pakyow-support/lib/pakyow/support/defineable.rb

```diff
@@ -61,6 +82,12 @@ def freeze
         super
       end
 
+      # Provide default make method
+      #
+      def make(*args, &block)
```

Comment:
I was thinking we wouldn't need this, but maybe you have some ideas. Can you elaborate?